sympy/combinatorics/pc_groups.py

```diff
@@ -0,0 +1,441 @@
+from sympy.core import Basic
+from sympy import sieve
+from sympy.combinatorics.perm_groups import PermutationGroup
+from sympy.printing.defaults import DefaultPrinting
+
+class PolycyclicGroup(DefaultPrinting):
+
+    is_group = True
+    is_solvable = True
+
+    def __init__(self, pc_sequence, pc_series):
+        self.pcgs = pc_sequence
+        self.pc_series = pc_series
+
+    def relative_order(self):
+        rel_orders = []
+        for i in range(len(self.pc_series)-1):
+            G = self.pc_series[i]
+            H = self.pc_series[i+1]
+            rel_orders.append(G.order() // H.order())
+        return rel_orders
+
+    def is_prime_order(self):
+        for order in self.relative_order():
+            if order not in sieve:
```

Comment:
How does `isprime` compare with `sieve` (like `return all(isprime(order) for order in self.relative_order())`)?

Reply:
For, greater length of relative_order `isprime` seems a good option.